api/middleware/jwt.go

```diff
@@ -31,16 +31,10 @@ func JwtConfigGenerate(jwtKey, realmName string, db *gorm.DB, logger *log.Logger
 				return userId, false
 			}
 			if !validLogin {
-				logger.WithFields(log.Fields{
-					"service": "api",
-					"user":    userId,
-				}).Error("bad login")
+				l.With("user", userId).Error("bad login")
```

Comment:
this should just be (with the above change):

```suggestion
				l.Error("bad login")
```